### PR TITLE
Add via metal volume

### PR DESCRIPTION
**Pull request by svandenb-dev**
compute via metal volume inside the hole for current density evaluation

### CODE REVIEW

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 1]
```diff
@@ -1394,6 +1394,39 @@ def name(self, value):
...
+        if not self.start_layer == self.stop_layer:
+            start_layer = self.start_layer
+            stop_layer = self.stop_layer
+            if self.backdrill_top:  # pragma no cover
+                start_layer = self.backdrill_top[0]
+            if self.backdrill_bottom:  # pragma no cover
+                stop_layer = self.backdrill_bottom[0]
+            padstack_def = self._pedb.padstacks.definitions[self.padstack_definition]
+            hole_diam = 0
+            try:  # pragma no cover
```
[the remark this answers is not in the document]
Comment (svandenb-dev) [reply pyaedt/edb_core/edb_data/padstacks_data.py:1417]: pads but here it's more to init the variable

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 2]
```diff
@@ -1394,6 +1394,39 @@ def name(self, value):
         self._edb_padstackinstance.SetName(value)
         self._edb_padstackinstance.SetProductProperty(self._pedb.edb_api.ProductId.Designer, 11, value)
 
+    @property
+    def metal_volume(self):
+        """Return the via hole instance metal volume in m3. Metal plating ratio is accounted.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:1399]:
```suggestion
        """Metal volume of the via hole instance in cubic units (m3). Metal plating ratio is accounted.
```
Property description should be in as a "noun" not a "verb." What is meant by this: "Metal plating ratio is accounted."? can it be rephrased as "The metal plating ratio is accounted for in the metal volume."?

File: pyaedt/edb_core/edb_data/padstacks_data.py [hunk 3]
```diff
@@ -1394,6 +1394,39 @@ def name(self, value):
...
+
+        Returns
+        -------
+        float
+            via hole instance metal volume.
```
Comment (PipKat) [inline pyaedt/edb_core/edb_data/padstacks_data.py:1404]:
```suggestion
            Metal volume of the via hole instance.
```

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**